src/capi/azext_capi/custom.py

```diff
@@ -460,12 +460,14 @@ def create_workload_cluster(  # pylint: disable=too-many-arguments,too-many-loca
         user_provided_template=None,
         bootstrap_commands=None,
         yes=False,
-        wait_for_nodes=False,
+        wait_for_nodes=1,
```

Comment:
Should this be 0 if `wait_for_nodes` is not specified?

Reply:
The behavior has been to wait for at least one node to be ready even before this flag, so I think this is backward-compatible.

Reply:
Ah that makes sense. I was also a bit concerned with there being an unnecessary check for if `wait_for_nodes > 0` but I guess a user can specify a value under 1.

Reply:
I think the idea is that having one Node ready is the minimal definition of a working cluster, so that's the least we can do before exiting.

You're right, if you want to skip even that check you can say `--wait-for-nodes=0`.